### PR TITLE
Add data-testids 

### PR DESCRIPTION
**Pull request by cornshuckin2**
Our Cypress suite was expecting these ids but appears they were gone after extracting this to a gem. Adding the missing id's back

### CODE REVIEW

File: app/components/anchor/autocomplete_component.html.erb [hunk 1]
```diff
@@ -24,6 +24,7 @@
   </div>
   <ul
     data-autocomplete-target="results"
+    data-testid="searchResultsList"
```
Comment (tysongach) [inline app/components/anchor/autocomplete_component.html.erb:27]: These test IDs, particularly this one, feel specific to a particular app, which makes me think they shouldn’t go here in the library, but instead added in when the component is used. Thoughts?

Looks like we’d need to add support in the component for doing that, but that wouldn’t be hard.

**Review by tysongach — COMMENTED** [review]
Shoot! This makes me wonder: why wasn’t there a failing test somewhere?

File: app/components/anchor/autocomplete_component.html.erb (hunk 1 above)
Comment (clawhun) [inline app/components/anchor/autocomplete_component.html.erb:27]: Would data-testid="search-results-list" be better to match the format of existing data testids?

**Comment by cornshuckin2** [comment]
> Shoot! This makes me wonder: why wasn’t there a failing test somewhere?

We run the Cypress suite on a schedule vs on every PR

File: app/components/anchor/autocomplete_component.html.erb (hunk 1 above)
Comment (cornshuckin2) [reply app/components/anchor/autocomplete_component.html.erb:27]: I agree! TBH I'm not familiar with adding it in the component if you could add an example

Comment (cornshuckin2) [reply app/components/anchor/autocomplete_component.html.erb:27]: Agreed! I just copy/pasta'd from what was there before

Comment (tysongach) [reply app/components/anchor/autocomplete_component.html.erb:27]: I’ve opened https://github.com/BuoySoftware/anchor_view_components/pull/12 to allow data attributes to be passed into the Loading Indicator component, and for the list slot in the Autocomplete component.

You’d pass them in like so:

```erb
<%= render Anchor::AutocompleteComponent.new(list_data: { testid: "searchResultsList" }) %>
```

```erb
<%= render Anchor::LoadingIndicatorComponent.new(data: { testid: "loading-indicator" }) %>
```

**Comment by cornshuckin2** [comment]
Closing in favor of https://github.com/BuoySoftware/anchor_view_components/pull/12

**Pull request closed without merging**